### PR TITLE
webgl: Fix incorrect hooks

### PR DESCRIPTION
**Pull request by chouquette**
This MR fixes a few hooks when enabling webgl calls tracing.

`bufferData` and `uniformMatrixXvf` take a different number of parameters depending on the webgl version being used. Without these patches the webgl1 flavor ends up being forced, which causes trouble since the parameters end up being shifted to different positions

### CODE REVIEW

File: src/library_webgl.js [hunk 1]
```diff
@@ -569,6 +569,17 @@ var LibraryGL = {
         var ret = (a9 !== undefined) ? glCtx['real_texSubImage3D'](a1, a2, a3, a4, a5, a6, a7, a8, a9, a10, a11) : glCtx['real_texSubImage2D'](a1, a2, a3, a4, a5, a6, a7, a8);
         return ret;
       };
+      glCtx['bufferData'] = function(a1, a2, a3, a4, a5) {
+          var ret = (a4 !== undefined) ? glCtx['real_bufferData'](a1, a2, a3, a4, a5) : glCtx['real_bufferData'](a1, a2, a3);
```
Comment (kripken) [inline src/library_webgl.js:573]:
```suggestion
          // WebGL1/2 versions have different parameters (not just extra ones)
          var ret = (a4 !== undefined) ? glCtx['real_bufferData'](a1, a2, a3, a4, a5) : glCtx['real_bufferData'](a1, a2, a3);
```

Comment (kripken) [reply src/library_webgl.js:573]: (this helps clarify why a different approach is needed in the case below)

**Review by kripken — APPROVED** [review]
Thanks! This looks right to me.

**Comment by chouquette** [comment]
Suggestion applied, thanks for your review! Let me know if you'd prefer me to squash it into the previous commit of if you'll squash all changes anyway!

**Comment by kripken** [comment]
The firefox test errors can be ignored here (sorry about that, still figuring it out), but the `other` error looks real - perhaps code size really increased here after opts.

It may be unavoidable, as this is a correctness fix. If so, running the failing test with `--rebaseline` locally will update the results properly.

@juj maybe you have an idea for avoiding a code size change here?

**Comment by chouquette** [comment]
Should be fixed by rebasing over f36f9fcaf83db93e6a6d0f0cdc47ab6379ade139

**Comment by kripken** [comment]
Thanks!

**Pull request merged**